docs/api/menu.md

```diff
@@ -15,7 +15,8 @@ the user right clicks the page:
 ```html
 <!-- index.html -->
 <script>
-const { Menu, MenuItem } = require('electron').remote;
+const remote = require('electron').remote;
```

Comment:
Would you mind making this `const { remote } = require('electron');` for consistency.